ckks/evaluator.go

```diff
@@ -1260,6 +1261,7 @@ func (eval *evaluator) MulAndAdd(op0, op1 Operand, ctOut *Ciphertext) {
 // The procedure will panic if either op0.Degree or op1.Degree > 1.
 // The procedure will panic if ctOut.Degree != op0.Degree + op1.Degree.
 // The procedure will panic if the evaluator was not created with an relinearization key.
+// The procedure will panic if ctOut = op0 or op1.
```

Comment:
Is it really necessary ? For MulRelinAndAdd the inputs and outputs ciphertexts could be of the same degree.